.all-contributorsrc

```diff
@@ -113,6 +113,13 @@
       "contributions": [
         "tool"
       ]
+    },
+    {
+      "login": "mgmcdermott",
+      "name": "Michael McDermott",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/8161781?v=3",
+      "profile": "http://mgmcdermott.com",
+      "contributions": []
```

Comment:
Could you add `"code"` to this? And then run `npm start gen` to regenerate the `README.md`